### ISSUE TITLE
How to I use customer tensorflow model with flutter_vision package ?

### ISSUE THREAD

**Issue by baimamboukar** [vladiH/flutter_vision#6]
#### I am working on a flutter project that requires some machine learning capabilities. I have my model already trained and ready to be used. 

I'm stuck at the level of integrating it to flutter. I have tried couple of packages that I found on pub.dev, but unfortunately I did not worked. Most of these packages are no longer maintained.

### How do I use this package to interact with customer tensorflow lite model ?

The README was not clear about that!

**Comment by vladiH** [comment]
Hi, this package is about only for managing YoloV5 model, if You have a model based on this, then It's for you

**Comment by krammnic** [comment]
It seems to me that for good recognition accuracy, one YOLO is not always enough.

**Comment by vladiH** [comment]
As I said earlier, this is just to manage YOLO and if you have a different model, then you should make your own implementation because there aren't many packages that can do this.

**Comment by baimamboukar** [comment]
> Hi, this package is about only for managing YoloV5 model, if You have a model based on this, then It's for you

Ooh! I understand now 😄

**Comment by krammnic** [comment]
@vladiH Can you make a template for integrating other neural network architectures in flutter_vision?

**Comment by vladiH** [comment]
@krammnic  Maybe, but not now. I'm thinking the hardest part to make a template is the preprocessing phase because it is different on each project, if you do it in flutter you will lose speed drastically.
